### PR TITLE
Release 1.1.0.

### PR DESCRIPTION
**Pull request by davidmorgan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 1.1.0
+
+- Moved `analsys_options.yaml` under `lib` so you can import it directly from
```
Comment (bwilkerson) [inline CHANGELOG.md]: "analsys" --> "analysis"

**Comment by a14n** [comment]
Did you push 1.1.0 on pub? I can't get it.

**Comment by davidmorgan** [comment]
Whoops. Now pushed. Thanks!

**Comment by a14n** [comment]
No problem ;)

**Pull request merged**